src/components/PeopleList.js

```diff
@@ -4,19 +4,22 @@ import { ListItem, Link } from 'react-toolbox';
 import { AutoSizer, List as InfiniteList } from 'react-virtualized';
 
 import { computeLegend } from '../services/Email';
+import { buildVCard } from '../services/People';
 import itemStyle from './item.scss';
 
 class PeopleList extends Component {
   itemRenderer = (someone) => {
     const phoneCallToAction = <Link href={`tel:${someone.phone}`} icon="phone" theme={itemStyle} />;
     const legend = computeLegend(someone, this.props.companyEmails);
+    const onItemClick = this.exportVcard.bind(this, someone, this.props.companyEmails);
     return (
       <ListItem
         key={someone.name}
         avatar={someone.avatar}
         caption={someone.name}
         legend={legend}
         rightIcon={phoneCallToAction}
+        onClick={onItemClick}
```

Comment:
onClick={() => exportVcard(someone, this.props.companyEmails)}

Reply:
So that you can remove the above binding

Reply:
OK, won't help performance though